platform/solo5/rumpukvm_bio.c

```diff
@@ -82,16 +91,20 @@ rumpuser_close(int fd)
 int
 rumpuser_getfileinfo(const char *name, uint64_t *size, int *type)
 {
-        struct solo5_block_info bi;
+	solo5_result_t ret;
 	int num;
 
 	if (bmk_strncmp(name, XENBLK_MAGIC, sizeof(XENBLK_MAGIC)-1) != 0)
 		return -1;
 	if ((num = devname2num(name)) == -1)
 		return BMK_ENXIO;
 
-        solo5_block_info(&bi);
-        *size = bi.capacity;
+	if (blk_handle == 0) {
+		ret = solo5_block_acquire(SOLO5_BLK_NAME, &blk_handle, &blk_info);
+		if (ret != SOLO5_R_OK)
+			return -1;
+    }
```

Comment:
nit: misaligned `}`

Reply:
It was my bad. Fixed with c71a670a1ddb6d7f998eedde6563ba57b3b7b5d4.